cypress/e2e/contactForm.cy.js

```diff
@@ -1,16 +1,16 @@
 import ContactFormPageObject from '../support/pages/contactForm.pageObject';
 import HomeAndCataloguePageObject
```

Comment:
There is a missing semicolon at the end of the import statement for HomeAndCataloguePageObject. Ensure that all import statements are properly terminated with a semicolon.